README.md

```diff
@@ -342,61 +346,54 @@ Please, make sure to install the following software/packages on the VMs.
 - Node v16.17.1 (only _host_)
 
   ```bash
-  curl https://raw.githubusercontent.com/creationix/nvm/master/install.sh | bash
-  nvm install 16.17.1
+  curl https://raw.githubusercontent.com/creationix/nvm/master/install.sh | bash && nvm install 16.17.1
   ```
 
 - Npm (only _host_)
 
   ```bash
-  sudo apt update
-  sudo apt install nodejs npm
+  sudo apt update --yes && sudo apt install --yes npm
   ```
 
 - Python 2 (only _host_)
 
   ```bash
-  sudo apt install python2 && alias python="/usr/bin/python2
+  sudo apt install python2 --yes && alias python="/usr/bin/python2"
   ```
 
-- Solc v0.5.16 (only _host_)
-
-  ```bash
-  sudo snap install solc
-  ```
+- Solc (solidity compiler) and Ganache CLI (only _host_)
 
-- Ganache CLI (only _host_)
   ```bash
-  sudo npm install -g ganache
+  npm install --global solc ganache
   ```
 
 ### Usage
 
 On the _host_ machine, please run
 
 ```bash
-cd ~
-git clone https://github.com/maticnetwork/matic-cli.git
-cd matic-cli
-npm i
-mkdir devnet
-cd devnet
+cd \
+  && git clone https://github.com/maticnetwork/matic-cli.git \
+  && cd matic-cli \
+  && npm install
 ```
 
+Set up this handy alias: `alias matic-cli="$(pwd)/bin/matic-cli"`.
+
 #### Local dockerized network
 
 Adjust the [docker configs](configs/devnet/docker-setup-config.yaml) and run
 
 ```bash
-../bin/matic-cli setup devnet -c ../configs/devnet/docker-setup-config.yaml
+mkdir devnet \
+  && cd devnet \
+  && matic-cli setup devnet --config ../configs/devnet/docker-setup-config.yaml | tee setup.log
+...
+DONE Devnet is ready
 ```
 
 Once the setup is done, follow these steps for local docker deployment
 
```

Comment:
aren't the script available under `matic-cli/devnet`? The `cd` to that path has been removed

Reply:
I removed it since we already move to `devnet` in the previous steps.